### ISSUE TITLE
[问题/Issue] 章节14：hello-agents库缺少两个类

### ISSUE THREAD

**Issue by kerwinqil** [datawhalechina/hello-agents#446]
### 1. 遇到问题的章节 / Affected Chapter

Chapter14

### 2. 问题类型 / Issue Type

依赖问题 / Dependency Issue

### 3. 具体问题描述 / Problem Description

<img width="759" height="128" alt="Image" src="https://github.com/user-attachments/assets/81221649-d38d-477e-9198-bcff8f59e4a6" />
安装了hello-agents库，但是缺少图片上的类

### 4. 问题重现材料 / Reproduction Materials

Package                      Version
---------------------------- -----------
absl-py                      2.3.0
aiofiles                     23.2.1
annotated-types              0.7.0
anyio                        4.5.2
astunparse                   1.6.3
brotlipy                     0.7.0
cachetools                   5.5.2
certifi                      2020.6.20
cffi                         1.15.1
charset-normalizer           2.0.4
click                        8.1.8
colorama                     0.4.5
conda                        22.11.1
conda-content-trust          0.1.3
conda-package-handling       1.9.0
contourpy                    1.1.1
cryptography                 38.0.1
cycler                       0.12.1
diskcache                    5.6.3
distro                       1.9.0
docker                       7.1.0
exceptiongroup               1.2.2
fastapi                      0.115.11
ffmpy                        0.5.0
filelock                     3.16.1
FLAML                        2.3.4
flatbuffers                  25.2.10
fonttools                    4.56.0
fsspec                       2025.2.0
gast                         0.4.0
google-auth                  2.40.3
google-auth-oauthlib         1.0.0
google-pasta                 0.2.0
google-search-results        2.4.2
gradio                       4.44.1
gradio_client                1.3.0
grpcio                       1.70.0
h11                          0.14.0
h5py                         3.11.0
httpcore                     1.0.7
httpx                        0.28.1
huggingface-hub              0.29.1
idna                         3.4
imbalanced-learn             0.12.4
importlib_metadata           8.5.0
importlib_resources          6.4.5
jieba                        0.42.1
Jinja2                       3.1.5
jiter                        0.9.1
joblib                       1.4.2
keras                        2.13.1
kiwisolver                   1.4.7
libclang                     18.1.1
Markdown                     3.7
markdown-it-py               3.0.0
MarkupSafe                   2.1.5
matplotlib                   3.7.5
mdurl                        0.1.2
menuinst                     1.4.19
mpmath                       1.3.0
mysql-connector-python       9.0.0
networkx                     3.1
numpy                        1.24.3
oauthlib                     3.2.2
openai                       2.2.0
opencv-python                4.11.0.86
opt_einsum                   3.4.0
orjson                       3.10.15
packaging                    24.2
pandas                       2.0.3
pdf2image                    1.17.0
pillow                       10.4.0
pip                          22.3.1
pluggy                       1.0.0
protobuf                     4.25.8
pyasn1                       0.6.1
pyasn1_modules               0.4.2
pycosat                      0.6.4
pycparser                    2.21
pydantic                     2.10.6
pydantic_core                2.27.2
pydub                        0.25.1
Pygments                     2.19.1
PyMuPDF                      1.24.11
pyOpenSSL                    22.0.0
pyparsing                    3.1.4
PyQt5                        5.15.4
pyqt5-plugins                5.15.4.2.2
PyQt5-Qt5                    5.15.2
PyQt5_sip                    12.15.0
pyqt5-tools                  5.15.4.3.1
PySocks                      1.7.1
pytesseract                  0.3.13
python-dateutil              2.9.0.post0
python-dotenv                1.0.1
python-multipart             0.0.20
pytz                         2025.1
pywin32                      311
PyYAML                       6.0.2
qt5-applications             5.15.2.2.3
qt5-tools                    5.15.2.1.3
regex                        2024.11.6
requests                     2.32.3
requests-oauthlib            2.0.0
rich                         13.9.4
rsa                          4.9.1
ruamel.yaml                  0.17.21
ruamel.yaml.clib             0.2.6
ruff                         0.9.9
safetensors                  0.5.3
scikit-learn                 1.3.2
scipy                        1.10.1
semantic-version             2.10.0
setuptools                   65.5.0
shellingham                  1.5.4
six                          1.16.0
sniffio                      1.3.1
starlette                    0.44.0
sympy                        1.13.3
tensorboard                  2.13.0
tensorboard-data-server      0.7.2
tensorflow                   2.13.0
tensorflow-estimator         2.13.0
tensorflow-intel             2.13.0
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.4.0
threadpoolctl                3.5.0
tiktoken                     0.7.0
tokenizers                   0.20.3
tomlkit                      0.12.0
toolz                        0.12.0
Werkzeug                     3.0.6
wheel                        0.37.1
win-inet-pton                1.1.0
wincertstore                 0.2
wrapt                        1.17.2
zipp                         3.20.2

 python --version
Python 3.11.3

### 确认事项 / Verification

- [x] 我已阅读过相关章节的文档 / I have read the relevant chapter documentation
- [x] 我已搜索过现有的Issues,确认此问题未被报告 / I have searched existing Issues and confirmed this hasn't been reported
- [x] 我已尝试过基本的故障排除(如重启、重新安装依赖等) / I have tried basic troubleshooting (restart, reinstall dependencies, etc.)

**Comment by zhao-11-yy** [comment]
兄弟一样的问题，请问解决了没有

**Comment by kerwinqil** [comment]
解决了，hello_agents用0.2.9版本



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年4月19日(周日) 晚上11:24
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;"State ***@***.***&gt;;
主题: Re: [datawhalechina/hello-agents] [问题/Issue] 章节14：hello-agents库缺少两个类 (Issue #446)


zhao-11-yy left a comment (datawhalechina/hello-agents#446)
 
兄弟一样的问题，请问解决了没有
